DSCircularCollectionView/Classes/DSCircularLayout.h

```diff
@@ -26,7 +26,7 @@
 
 @property (nonatomic, assign) BOOL rotateItems;
 
--(void)initWithCentre:(CGPoint)centre radius:(CGFloat)radius itemSize:(CGSize)itemSize andAngularSpacing:(CGFloat)angularSpacing;
+-(void)initWithCentre:(CGPoint)centre radius:(CGFloat)radius itemSize:(CGSize)itemSize andAngularSpacing:(CGFloat)angularSpacing isCountingWidth:(BOOL)isCountingWidth;
```

Comment:
Can you leave the previous init as is and add this as another init so that the existing users can still be supported. Also, make sure you set isCountingWidth to NO in the older init implementation.